qiskit/pulse/instructions/frequency.py

```diff
@@ -60,3 +60,33 @@ def channel(self) -> PulseChannel:
         scheduled on.
         """
         return self._channel
+
+
+class ShiftFrequency(Instruction):
+    """Shift the channel frequency away from the set frequency."""
+
+    def __init__(self, frequency: float,
```

Comment:
Could `frequency` be moved to its own line?

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/4390/commits/7ecf5ab6be4f22c58b889b1969bfba31b34a4545.